Lib/test/test_symtable.py

```diff
@@ -144,6 +144,16 @@ def test_annotated(self):
         self.assertTrue(st4.lookup('x').is_local())
         self.assertFalse(st4.lookup('x').is_annotated())
 
+        # Test that annotations in the global scope are valid after the
+        # variable is declared as nonlocal.
+        st5 = symtable.symtable('global x\nx: int', 'test', 'exec')
+        self.assertTrue(st5.lookup("x").is_global())
+
+        # Test that annotations for nonlocals are valid after the
+        # variable is declared as nonlocal.
+        st6 = symtable.symtable('def g():\n    x=2\n    def f():\n'
+                                '        nonlocal x\n    x:int', 'test', 'exec')
```

Comment:
Complex multi-line tests are easier to read if you split them like this:
```python
        st6 = symtable.symtable('def g():\n'
                                '    x = 2\n'
                                '    def f():\n'
                                '        nonlocal x\n'
                                '    x: int',
                                'test', 'exec')
```
(Also I would stick to PEP 8 even in tests, see other test above. In this case space around `=` and space after `:` in annotation.)

Reply:
Done in 547dbba. I would like to add an extra nonlocal check to `st6` after if #9872 is merged.